src/components/routes/HubManagerNav.jsx

```diff
@@ -1,19 +1,20 @@
-import React from 'react';
-import PrivateRoute from './PrivateRoute';
-import { Switch } from 'react-router-dom';
-import HubManDashboard from '../hubManagers/HubManDashboard';
-import Packages from '../Packages';
-import Invoice from '../Invoice';
-
-const HubManagerNav =()=>{
-    return (
-        <Switch>
-            <PrivateRoute exact path="/" component={HubManDashboard}/>
-			<PrivateRoute exact path="/packages" component={Packages} />
-			<PrivateRoute exact path="/invoice" component={Invoice} />
-		
-		</Switch>
-    )
-}
-
+
+import React from 'react';
+import PrivateRoute from './PrivateRoute';
+import { Switch } from 'react-router-dom';
+import HubManDashboard from '../hubManagers/HubManDashboard';
```

Comment:
Prefer usage of a more descriptive name
```suggestion
import HubManagerDashboard from '../hubManagers/HubManDashboard';
```